ui/console/src/main/scripts/plugins/metrics/plugins/metrics/html/app-details/detail-datasources.html

```diff
@@ -55,33 +55,55 @@ <h3 class="pull-left">Metrics</h3>
           <button class="btn btn-link hk-chart-update" tooltip-trigger tooltip-placement="top" tooltip="Update chart"><i class="fa fa-refresh"></i></button>
         </div>
       </div>
-      <div class="panel panel-default hk-graph">
-        <h2>Availability and Responsiveness</h2>
-        <!-- <img src="img/graph-av-rp.png" alt="graph-av-rp"> -->
+
+      <div class="panel panel-default hk-graph" ng-if="vm.chartAvailData[ds.id][2]">
+        <h2>Availability</h2>
+        <div class="metrics-chart">
+          <hawkular-chart
+              multi-data="{{vm.chartAvailData[ds.id]}}"
+              chart-type="multiline"
+              y-axis-units="bytes"
+              chart-height="250">
+          </hawkular-chart>
+        </div>
         <div class="row hk-legend hk-legend-inline text-left">
           <div class="col-md-7 col-lg-6">
-            <h4>Connection Availability<!-- <img src="img/legend-connection.png" alt="legend-connection" height="12"> --></h4>
+            <h4>Connection Availability</h4>
             <div class="hk-graphselector hk-blue">
-              <input type="checkbox" id="available1" checked="checked">
+              <input type="checkbox" checked="checked" disabled>
               <label for="available1"><span>Available Connections</span></label>
             </div>
             <div class="hk-graphselector hk-green">
-              <input type="checkbox" id="connections1">
+              <input type="checkbox" checked="checked" disabled>
               <label for="connections1"><span>Connections in Use</span></label>
             </div>
             <div class="hk-graphselector hk-gray">
-              <input type="checkbox" id="timed1">
+              <input type="checkbox" checked="checked" disabled>
               <label for="timed1"><span>Timed-out Connections</span></label>
             </div>
           </div>
-          <div class="col-md-5 col-lg-6">
-            <h4>Responsiveness<!-- <img src="img/legend-responsiveness.png" alt="legend-responsiveness" height="12"> --></h4>
+        </div>
+      </div>
+
+      <div class="panel panel-default hk-graph" ng-if="vm.chartRespData[ds.id][1]">
+        <h2>Responsiveness</h2>
+        <div class="metrics-chart">
+          <hawkular-chart
+              multi-data="{{vm.chartRespData[ds.id]}}"
+              chart-type="multiline"
```

Comment:
Don't recall we moving to gauge type for these charts, would they make sense for this data? This is for the expanded "View Metrics" showing pool availability (available/inuse/timedout connections) and responsiveness (conn wait and creation time), mocks use muiltiline: https://issues.jboss.org/secure/attachment/12390611/Datasources3-expanded2.jpg

I've used two different charts as our multiline is not ready for double y-axis units, so it would look odd as units scale differ a lot. The unit type "bytes" was forgotten, had it correct but undo-ed some changes to fix a problem and forgot about it. will fix.

We don't have the Gauge chart type ready for the connections used.